### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by atsen-dev** [ubuntu-flutter-community/settings#262] [General]
Hello,

I've heard that you need some Flutter contributers, I'm a Flutter developer (also at professionnal level) so I can contribute.

Did you always need contributors ? 

Thanks

**Comment by Feichtmeier** [comment]
hi @atsen-dev !

Thanks for wanting to help out - yes there are still quiet a bunch panels needed. Most of them required either working with GSettings.dart (easy - example https://github.com/Feichtmeier/settings/blob/master/lib/view/pages/appearance/dock_model.dart) or with dbus.dart (harder, but depends on the dbus signatures, example: https://github.com/Feichtmeier/settings/blob/master/lib/services/power_settings_service.dart)

If you feel like you have time for this, just grab one of the issues :) Thank you in advance!

**Reply by yash-garg** [reply]
Hey, just a quick question. Is there any specific Flutter & Ubuntu version on which you guys are working on?

Will try to pick something up from issues section!

**Reply by Feichtmeier** [reply]
> Hey, just a quick question. Is there any specific Flutter & Ubuntu version on which you guys are working on?

Thanks for the question. Sorry for not being more specific here. Previously I was using flutter master and Ubuntu 21.10
Now that flutter master has become a bit difficult to work with many deps and flutter stable works really well on Linux I switched flutter to stable. Also I switched my host to 22.04 to keep up with all the new dark mode changes. I should be more clear in the readme :)

**Reply by yash-garg** [reply]
Thank you for the quick response, that was helpful!

**Reply by atsen-dev** [reply]
@Feichtmeier  I think you should use a fixed version and update it yourself when you want, to avoid some compatibility issue if someone use a previous version and you an up-to-date version.

You should use [fvm](https://fvm.app/) for example.
